### DISCUSSION TITLE
其实仔细想了想，还是彻头彻尾的精英主义

### DISCUSSION THREAD

**Discussion by Senior-Jiang** [The-Run-Philosophy-Organization/run#177] [General]
比如把人民和羊群这个概念分出来，就一股味。只能说程序员谈起政治来必定有这种倾向。润就润吧，还要掉书袋，毕竟政治经济学内容一大堆，总能找到论证自己想法的内容。

**Comment by Senior-Jiang** [comment]
当然，我还是认为润的人越多越好，只要不搞间谍活动

**Comment by the-run-philosophy** [comment]
最可笑的是，布尔什维克本身就是最大的最极端的精英主义。。。。。

**Reply by leon-fudan** [reply]
是吗？你们赶紧run吧，中国GDP目前还只是美国的六分之一，当然养不起一堆摸鱼的程序员，好好去欧美摸鱼。祝福你们。
想努力的人，留下，我们一起建设祖国，好吗？ 我不敢说现在我们可以活的比美国程序员好。
但是，当我们人均GDP和美国可比的时候，我相信这些努力的人，会活的比美国同行好。

**Reply by the-run-philosophy** [reply]
确实，还是你这样愿意牺牲自己的人好。做一个永远服从，永远听话，永远感恩的___。
希望在您的思想被完全封锁封控禁锢的时候还能写出任何一行可以被人类使用的代码～

**Reply by leon-fudan** [reply]
抱歉，请别扣帽子，也不要诅咒想法和你不一样的人好吗？
我先说好，我和在中国的大多数人一样，生活是我自己工作和努力的结果，没有感恩谁，也没有服从谁。
同时思想和你一样open，甚至也支持你们run away，希望你也支持我，让我看看你的文明和素质，谢谢。

**Reply by charchar114514** [reply]
那我诚挚地祝福宁及家人生生世世永远轮回大中华🥰

**Reply by Xinrui-Wang** [reply]
> 抱歉，请别扣帽子，也不要诅咒想法和你不一样的人好吗？ 我先说好，我和在中国的大多数人一样，生活是我自己工作和努力的结果，没有感恩谁，也没有服从谁。 同时思想和你一样open，甚至也支持你们run away，希望你也支持我，让我看看你的文明和素质，谢谢。

”生活是你自己工作努力的结果“ 笑拉了

**Reply by Fontzs** [reply]
> 是吗？你们赶紧run吧，中国GDP目前还只是美国的六分之一，当然养不起一堆摸鱼的程序员，好好去欧美摸鱼。祝福你们。
> 想努力的人，留下，我们一起建设祖国，好吗？ 我不敢说现在我们可以活的比美国程序员好。
> 但是，当我们人均GDP和美国可比的时候，我相信这些努力的人，会活的比美国同行好。

把我给逗乐了

**Reply by mitchellcuevas** [reply]
> 

中国最后就一个大号朝鲜，还想比美国。笑拉了。

**Comment by yangxin9003** [comment]


**Reply by wubabalubab** [reply]
人类本身就是这样,境界低的想我追求物质满足,境界高德追求精神满足

**Reply by the-run-philosophy** [reply]
如果这个世界没有铁拳，你说润的人“精致利己”没错
如果这个世界有铁拳，润的人只是在”逃命“罢了。
如果你看不到铁拳的存在，我觉得你可以提高下自己的觉悟。多观察。
哪怕只是观察一下倒闭的企业，询问下原因是什么，理解一下什么叫”国进民退“和”布尔什维克“。
也可以
很难吗？谢谢。

**Reply by ytgui** [reply]
一口一个精致利己，看来层主已经买了10+套房，或者处级以上当着羊上羊。哪怕你996工作过一个月都会重新认识到：整天把精致利己挂在嘴边的才是真正的精致的利己

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
